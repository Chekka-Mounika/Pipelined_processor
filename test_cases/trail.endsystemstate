Hash of the Processor State = -1414219998
Number of instructions executed = 34
Number of cycles taken = 45
Number of wrong branches taken :8
Number of wrong instructions entered pipeline for wrong branches :16
Number of haults : 7
