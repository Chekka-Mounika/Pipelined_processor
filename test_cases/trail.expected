Hash of the Processor State = 255541867
Number of instructions executed = 365
Number of cycles taken = 475
Number of wrong branches taken :21
Number of wrong instructions entered pipeline for wrong branches :42
Number of haults : 106
